### PR TITLE
install and update

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: scripts/update.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-. "$PWD"/scripts/utils.sh
+."$PWD"/scripts/utils.sh
```
Comment (Kikobeats) [inline scripts/update.sh:3]: why delete the space?

Comment (ghost) [inline scripts/update.sh:3]:
```
uno-zen/scripts/update.sh: 3: .: Can't open /home/user/ghost/content/themes/scripts/utils.sh
```

with the space its no correct path

Comment (Kikobeats) [inline scripts/update.sh:3]: hmm maybe is necessary remove also in `install.sh`?

**Comment by ghost** [comment]
Will check it this evening...somethings wrong there

**Pull request closed without merging**